gate-web/src/main/groovy/com/netflix/spinnaker/gate/config/SlackConfigProperties.java

```diff
@@ -8,4 +8,5 @@
 public class SlackConfigProperties {
   String token;
   String baseUrl;
+  Long channelRefreshIntervalInMs;
```

Comment:
```suggestion
  Integer channelRefreshIntervalMillis;
```
That's more consistent with how we define properties